docs/dqx/docs/guide/quality_checks_apply.mdx

```diff
@@ -51,10 +51,11 @@ You can customize the reporting columns as described in the [additional configur
     from databricks.labs.dqx.engine import DQEngine
     from databricks.labs.dqx.rule import DQRowRule, DQDatasetRule, DQForEachColRule
     from databricks.labs.dqx.config import InputConfig, OutputConfig
+    from databricks.labs.dqx.metrics_observer import DQMetricsObserver
```

Comment:
It would be better ot have a separate example for this. User may get an impression that metrics needs to be always provided